pageserver/src/task_mgr.rs

```diff
@@ -1,4 +1,6 @@
 //!
+//! TODO need update comment / at least mention context.rs
```

Comment:
👍

Reply:
Left `task_mgr.rs` as is, since this patch doesn't change anything about it.

Added a large block comment to `context.rs`, though.